index.js

```diff
@@ -0,0 +1,69 @@
+const csv = require('csvtojson')
+const fs = require('promise-fs')
+
+const convertDKBJsonToYNABJson = (inputJson) => {
+  console.log('Log inputJason')
+  console.log(inputJson)
+  console.log('Number of input objects:' + inputJson.length)
+  let outputJson = []
+
+  const createYNABJsonMapper = (entry) => {
+    let outflow = 0
+    let inflow = 0
+    const moneyFlow = parseFloat(entry['Betrag (EUR)'])
+    if (moneyFlow < 0) {
+      outflow = String(-1 * moneyFlow)
+      inflow = ''
+    } else if (moneyFlow > 0) {
+      outflow = ''
+      inflow = String(moneyFlow)
+    }
+
+    outputJson.push({
+      Date: entry.Buchungstag,
+      Payee: entry['Auftraggeber / Beg�nstigter'],
+      Memo: entry.Verwendungszweck,
+      Outflow: outflow,
+      Inflow: inflow,
+    })
+  }
+  // inlcude map
+  // include mapper function
+  // define mapper in global
+  // test mapper in edge cases
+
+  inputJson.map(createYNABJsonMapper)
```

Comment:
Nice. You iterate through the `inputJson` Array and create a new Array with the mapped values. And then you do nothing with this new Array but forget about it and return the old one in the next line.
```suggestion
  return inputJson.map(createYNABJsonMapper)
```